src/theme/style.scss

```diff
@@ -1,29 +1,81 @@
 @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap");
+$mycolor: "#83CE27";
 
 body {
-  font-family: "Montserrat", "sans-serif";
+  font-family: "serif", "sans-serif";
+  background-color: "#83CE27";
 }
 
+svg{
+ display: flex;
+ position: relative;
+}
 .title {
   font-size: 2.25em;
+  font-weight: bold;
+  letter-spacing: 2px;
   text-transform: uppercase;
 }
 
-.welcome-content {
+#root{
+  margin: 0px;
+  padding: 0px;
+}
+
+ion-button.skip{
+  margin-top: 10%;
+  margin-left: 3%;
+  width: 35%;
+  height: 50px;
+  font-size: 1.5em;
+}
+
+.star{
+  position: absolute;
```

Comment:
This probably shouldn't be absolutely positioned, it should be able to fit into the page layout somehow.